modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationRegistry.java

```diff
@@ -436,4 +465,28 @@ private void checkPolymorphicConfigIds(Map<Class<?>, Set<Class<?>>> polymorphicE
             ids.clear();
         }
     }
+
+    private Collection<CompletableFuture<?>> notifyStorageRevisionListeners(long oldStorageRevision, long newStorageRevision) {
+        if (oldStorageRevision == newStorageRevision || storageRevisionListeners.isEmpty()) {
```

Comment:
I don't think that `oldStorageRevision` is really required here

Reply:
Removed it.